Canada data up

cardanarbo opened the pull request:
Canada data now displays correctly on the site. (A minor bug fix to the mouseover text, as well.)

cardanarbo committed:
Pulling in information for a Canadian data source

Initial setup; untested. (Also contains minor documentation fixes on the Illinois dataset.)

cardanarbo committed:
Added Canadian map

Plus bug fixes

cardanarbo committed:
Merge remote-tracking branch 'origin/Sarah' into canada

cardanarbo committed:
Fixed up Canadian data

Canadian data now displays on the site

geobarry reviewed (changes requested):
Looks good. Can you please merge the master branch into your branch and resolve any conflicts? Also please describe the mouseover bug you fixed. Because I've found a bug I was planning to fix, I want to know if it's the same one.

cardanarbo committed:
Created .gitignore file

Stashed my IDE files in .gitignore

cardanarbo committed:
Merge branch 'master' into canada

cardanarbo committed:
Added proper credit for the Canadian data

cardanarbo commented:
The mouseover text was displaying the incorrect date - it was displaying as the day as the day of the week, not as the day of the month.

(The branch is merged! I also added proper credits for the Canadian data, as well as a .gitignore file)

Commit:
Fixed Legend Bugs
- Fixed issue that legend did not update after animation
- Set legend to update only when user clicks on a time period, but legend is fixed when navigation time using arrow keys.

geobarry commented:
> The mouseover text was displaying the incorrect date - it was displaying as the day as the day of the week, not as the day of the month.
> 
> (The branch is merged! I also added proper credits for the Canadian data, as well as a .gitignore file)

OK good! I had noticed the odd behavior and had it on my "to-do list" to figure out what was going on, but hadn't figured it out yet. Thanks for fixing that!
